packages/blockprotocol/core.d.ts

```diff
@@ -62,7 +62,7 @@ export type BlockMetadata = {
   /**
    * The dependencies a block relies on but expects the embedding application to provide
    */
-  externals: {
+  externals?: {
```

Comment:
Adding `?` fixed a TSC issue for me around `blocksData as BlockMetadata[]` in `site/src/pages/api/blocks.api.ts`. `blocksData` comes from `site/blocks-data.json`, where some elements don’t have `externals`. This error does not show up in CI because `blocksData` is an empty array when blocks are not built.